pullrequest.rst

```diff
@@ -119,10 +119,27 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you can add it to the list of changes to be committed (the "staging area")::
+
+   git add <file>
```

Comment:
I've add the chect sheet and git-scm <progit> to the end.

And remove the last line about staging area, but still reserve most word, I think we still need to provide a clear message to contributer.